src/components/tile-select/tile-select.spec.js

```diff
@@ -165,6 +176,24 @@ describe("TileSelect", () => {
     );
   });
 
+  it("renders group description element as p when description prop is passed as string", () => {
+    renderWithTitleSelectGroup({ description: "description" });
+    expect(wrapper.find(StyledGroupDescription).prop("as")).toBe(undefined);
+    expect(wrapper.find(StyledGroupDescription).prop("children")).toBe(
+      "description"
+    );
+  });
+
+  it("does not render group description element, as description prop is undefined", () => {
+    renderWithTitleSelectGroup();
+    expect(wrapper.find(StyledGroupDescription).exists()).toEqual(false);
+  });
+
+  it("does not render group description element, as description prop is empty", () => {
```

Comment:
The undefined and empty string tests are essentially the same thing as both are falsy values, it's okay to just have one or the other imo